packages/core/src/test/shared/clients/dynamoDbClient.test.ts

```diff
@@ -0,0 +1,154 @@
+        })
+
+        it('should handle an AWSError correctly', async () => {
+            const request = { TableName: 'Table1' }
+
+            sinon.stub(dynamoDbClient, 'getTableInformation').rejects()
+
+            try {
+                await dynamoDbClient.getTableInformation(request)
+                assert.fail('Expected error not thrown')
+            } catch (err: any) {
+                assert.equal(err.message, 'Error')
+            }
+        })
+    })
+
+    describe('scanTable', function () {
+        it('should retrieve the items from a table', async () => {
+            const request = { TableName: 'Table1' }
+            const expectedResult = { Items: { data: 'value' } } as unknown as ScanOutput
+
+            sinon.stub(dynamoDbClient, 'scanTable').resolves(generateRequest(expectedResult))
+            const actualResult = await dynamoDbClient.scanTable(request)
+            assert.deepStrictEqual(expectedResult, actualResult)
+        })
+
+        it('should handle an AWSError correctly', async () => {
+            const request = { TableName: 'Table1' }
+
+            sinon.stub(dynamoDbClient, 'scanTable').rejects()
+
+            try {
+                await dynamoDbClient.scanTable(request)
+                assert.fail('Expected error not thrown')
+            } catch (err: any) {
+                assert.equal(err.message, 'Error')
+            }
+        })
+    })
+
+    describe('deleteTable', function () {
+        it('should be able to delete a table', async () => {
+            const request = { TableName: 'Table1' }
+            const expectedResult = { TableDescription: { TableName: 'Table1' } } as unknown as DeleteTableOutput
+
+            sinon.stub(dynamoDbClient, 'deleteTable').resolves(generateRequest(expectedResult))
+            const actualResult = await dynamoDbClient.deleteTable(request)
+            assert.deepStrictEqual(expectedResult, actualResult)
+        })
+
+        it('should handle an AWSError correctly', async () => {
+            const request = { TableName: 'Table1' }
+
+            sinon.stub(dynamoDbClient, 'deleteTable').rejects()
+
+            try {
+                await dynamoDbClient.deleteTable(request)
+                assert.fail('Expected error not thrown')
+            } catch (err: any) {
+                assert.equal(err.message, 'Error')
+            }
+        })
+    })
+
+    describe('queryTable', function () {
+        it('should be able to query data from a table', async () => {
+            const request = { TableName: 'Table1' }
+            const expectedResult = { Items: { data: 'value' } } as unknown as QueryOutput
+
+            sinon.stub(dynamoDbClient, 'queryTable').resolves(generateRequest(expectedResult))
+            const actualResult = await dynamoDbClient.queryTable(request)
+            assert.deepStrictEqual(expectedResult, actualResult)
+        })
+
+        it('should handle an AWSError correctly', async () => {
```

Comment:
```suggestion
        it('should handle an AWSError', async () => {
```
I think a couple of the test names can be simplified. They should just convey the minimum amount of information needed to tell what the test is doing. We don't need extra words there since they make it harder to parse

Reply:
This also applies for the other test names that I haven't commented on as well